exoctk/exoctk_app/form_validation.py

```diff
@@ -46,7 +46,7 @@ class FortneyModelForm(BaseForm):
 class LimbDarkeningForm(BaseForm):
     """Form validation for the limb_darkening tool"""
     # Model grid
-    default_modelgrid = os.path.join(MODELGRID_DIR, 'ATLAS9/')
+    default_modelgrid = os.path.join(MODELGRID_DIR, 'default/')
```

Comment:
@mfixstsci There was an issue with the data where some of these directories were missing, and the temporary but wrong solution was to change them all to `default`. Is this one of those or should this actually go from `ATLAS9` to `default`?

Reply:
@mfixstsci I actually just answered this question myself while monkeying around with out data. 
Please change this back to `ATLAS9` -- there are no `ATLAS` models in `default`, it only contains a smaller subset of the `ACES` data.

Reply:
@julesfowler sounds good!